server/utilities/PokerUtils.js

```diff
@@ -30,7 +30,6 @@ const NUMBER_OF_SUITS = 4;
  * An array of poker table stats
  */
 const POKER_TABLE_STATS = [
-    { name: 'Frugal', bigBlind: 1, buyIn: 10 },
```

Comment:
Why is this being removed?

Reply:
It needs to be removed, it's an item on Trello. We end up with 0.5 chips.

Reply:
Ah, I forgot about that